<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>653</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe Print</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Image Editor (OpenCV)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Icons/Icons/brightness(1).png</normaloff>:/Icons/Icons/brightness(1).png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(200, 198, 199)</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTabWidget" name="Tab">
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <widget class="QWidget" name="EditTab">
         <property name="font">
          <font>
           <family>Segoe Print</family>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <attribute name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/Icons/Icons/image.png</normaloff>:/Icons/Icons/image.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Upload and Edit</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QPushButton" name="Brightness">
               <property name="font">
                <font>
                 <family>Segoe Print</family>
                 <pointsize>7</pointsize>
                 <weight>75</weight>
                 <italic>false</italic>
                 <bold>true</bold>
                 <underline>false</underline>
                 <strikeout>false</strikeout>
                 <kerning>false</kerning>
                </font>
               </property>
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="tabletTracking">
                <bool>true</bool>
               </property>
               <property name="focusPolicy">
                <enum>Qt::WheelFocus</enum>
               </property>
               <property name="acceptDrops">
                <bool>true</bool>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(186, 189, 182);</string>
               </property>
               <property name="text">
                <string>Brightness</string>
               </property>
               <property name="icon">
                <iconset theme="brightness" resource="resources.qrc">
                 <normaloff>:/Icons/Icons/brightness(1).png</normaloff>:/Icons/Icons/brightness(1).png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>57</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Contrast">
               <property name="font">
                <font>
                 <family>Segoe Print</family>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(186, 189, 182);</string>
               </property>
               <property name="text">
                <string>Contrast</string>
               </property>
               <property name="icon">
                <iconset theme="brightness" resource="resources.qrc">
                 <normaloff>:/Icon/Icons/brightness(2).png</normaloff>:/Icon/Icons/brightness(2).png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>57</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Color">
               <property name="font">
                <font>
                 <family>Segoe Print</family>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(186, 189, 182);</string>
               </property>
               <property name="text">
                <string>Color</string>
               </property>
               <property name="icon">
                <iconset theme="brightness" resource="resources.qrc">
                 <normaloff>:/Icons/Icons/color-circle.png</normaloff>:/Icons/Icons/color-circle.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>57</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Hue">
               <property name="font">
                <font>
                 <family>Segoe Print</family>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(186, 189, 182);</string>
               </property>
               <property name="text">
                <string>Hue</string>
               </property>
               <property name="icon">
                <iconset theme="brightness" resource="resources.qrc">
                 <normaloff>:/Icons/Icons/hue-circle.png</normaloff>:/Icons/Icons/hue-circle.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>57</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Sharpness">
               <property name="font">
                <font>
                 <family>Segoe Print</family>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(186, 189, 182);</string>
               </property>
               <property name="text">
                <string>Sharpness</string>
               </property>
               <property name="icon">
                <iconset theme="brightness" resource="resources.qrc">
                 <normaloff>:/Icons/Icons/sharp.png</normaloff>:/Icons/Icons/sharp.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>57</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Resize">
               <property name="font">
                <font>
                 <family>Segoe Print</family>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(186, 189, 182);</string>
               </property>
               <property name="text">
                <string>Resize</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/Icons/Icons/expand.png</normaloff>:/Icons/Icons/expand.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>57</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Save">
               <property name="font">
                <font>
                 <family>Segoe Print</family>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(186, 189, 182);</string>
               </property>
               <property name="text">
                <string>Save</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/Icon/Icons/diskette.png</normaloff>:/Icon/Icons/diskette.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>57</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Exit">
               <property name="font">
                <font>
                 <family>Segoe Print</family>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(186, 189, 182);</string>
               </property>
               <property name="text">
                <string>Exit</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/Icon/Icons/cancel.png</normaloff>:/Icon/Icons/cancel.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QPushButton" name="Upload">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="font">
                <font>
                 <family>Segoe Print</family>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(186, 189, 182);</string>
               </property>
               <property name="text">
                <string>Gallery</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/Icons/Icons/image.png</normaloff>:/Icons/Icons/image.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="editlabel">
               <property name="font">
                <font>
                 <family>Segoe Print</family>
                 <pointsize>14</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="textFormat">
                <enum>Qt::PlainText</enum>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QGraphicsView" name="editView">
               <property name="autoFillBackground">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:rgb(255,255,255)</string>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <family>Nirmala UI</family>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QSlider" name="horizontalSlider">
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background:rgb(200, 198, 199)</string>
               </property>
               <property name="minimum">
                <number>-100</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="singleStep">
                <number>10</number>
               </property>
               <property name="pageStep">
                <number>10</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="sliderPosition">
                <number>0</number>
               </property>
               <property name="tracking">
                <bool>true</bool>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="invertedControls">
                <bool>false</bool>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksAbove</enum>
               </property>
               <property name="tickInterval">
                <number>10</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="OpenCamera">
         <attribute name="icon">
          <iconset>
           <disabledoff>:/Icons/Icons/camera.png</disabledoff>
          </iconset>
         </attribute>
         <attribute name="title">
          <string>Open Camera</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="1">
           <widget class="QScrollArea" name="CameraView">
            <property name="styleSheet">
             <string notr="true">background-color:rgb(255,255,255)</string>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>747</width>
               <height>465</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QPushButton" name="Camera">
            <property name="font">
             <font>
              <family>Segoe Print</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(186, 189, 182);</string>
            </property>
            <property name="text">
             <string>Capture</string>
            </property>
            <property name="icon">
             <iconset theme="brightness" resource="resources.qrc">
              <normaloff>:/Icons/Icons/camera.png</normaloff>:/Icons/Icons/camera.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
    <addaction name="actionUpload_Image"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionBrightess"/>
    <addaction name="actionContrast"/>
    <addaction name="actionColor"/>
    <addaction name="actionHue"/>
    <addaction name="actionSharpness"/>
    <addaction name="actionResize"/>
    <addaction name="separator"/>
    <addaction name="actionImage_Display"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icon/Icons/diskette.png</normaloff>:/Icon/Icons/diskette.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icon/Icons/cancel.png</normaloff>:/Icon/Icons/cancel.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionBrightess">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/brightness(1).png</normaloff>:/Icons/Icons/brightness(1).png</iconset>
   </property>
   <property name="text">
    <string>Brightess</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionContrast">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icon/Icons/brightness(2).png</normaloff>:/Icon/Icons/brightness(2).png</iconset>
   </property>
   <property name="text">
    <string>Contrast</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/color-circle.png</normaloff>:/Icons/Icons/color-circle.png</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionHue">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/hue-circle.png</normaloff>:/Icons/Icons/hue-circle.png</iconset>
   </property>
   <property name="text">
    <string>Hue</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="actionSharpness">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/sharp.png</normaloff>:/Icons/Icons/sharp.png</iconset>
   </property>
   <property name="text">
    <string>Sharpness</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionResize">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/expand.png</normaloff>:/Icons/Icons/expand.png</iconset>
   </property>
   <property name="text">
    <string>Resize</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionImage_Display">
   <property name="text">
    <string>Image_Display</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>&amp;End Full Screen</string>
   </property>
   <property name="toolTip">
    <string>Exit Screen</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionUpload_Image">
   <property name="text">
    <string>Upload</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>Tab</tabstop>
  <tabstop>Contrast</tabstop>
  <tabstop>Upload</tabstop>
  <tabstop>Save</tabstop>
  <tabstop>Exit</tabstop>
  <tabstop>Sharpness</tabstop>
  <tabstop>horizontalSlider</tabstop>
  <tabstop>Resize</tabstop>
  <tabstop>Hue</tabstop>
  <tabstop>CameraView</tabstop>
  <tabstop>Camera</tabstop>
  <tabstop>editView</tabstop>
  <tabstop>Color</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
